### ISSUE TITLE
Feature Request - graduated tempo change

### ISSUE THREAD

**Issue by kev-in-jp** [hydrogen-music/hydrogen#316]
It would be much more useful to be able to have a gradual tempo change available, say spread across one bar rather than a jump to a new tempo.

**Comment by trebmuh** [comment]
+1

**Comment by trebmuh** [comment]
+1, another one, from http://hydrogen.popez.org/hcms/node/16796

**Comment by SilasMortimer** [comment]
Yeah, that was my post. I would greatly appreciate this feature and I imagine a lot of people would use it if it was available. An additional consideration, as I mentioned in that post, might be an ability to shape the graduation of the change, maybe with a slider or something.

**Comment by youkii** [comment]
Now that there is tempo ramps in Ardour5, the possibility to link it with Hydrogen through Jack Transport would be great.

**Comment by trebmuh** [comment]
@youkii : isn't it already working ? (just asking here, because I don't know)
Ie: is ardour5 is selected as transport-master and hydrogen as transport-slave, then you make a tempo ramp in ardour, then start it all, Hydrogen wouldn't be supposed to follow the transport from ardour (which is a ramp) in that use-case ?

**Comment by youkii** [comment]
@trebmuh
Maybe i did something wrong, but when i tried it didn't work. That's the reason why i came to ask here.

**Comment by rekado** [comment]
@trebmuh I just tried this with Ardour 5 and Hydrogen 0.9.6.1.  Ardour is set up as the time master, Hydrogen's JACK transport mode is ON and JACK time master mode is OFF.

As the tempo ramps up in Ardour the tempo in Hydrogen is unchanged.  Once Ardour reaches the end of the tempo ramp, the tempo in Hydrogen steps up and the position in the track is synchronized after a jump.

**Comment by mevsme** [comment]
Even Ardour has this feature after dozens of days of development. I don't think it is easy to implement feature.
And still, you can not bind audio clip to change it's tempo while tempo changes like in Cubase or wherever.
